tests/parser/features/test_immutable.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+type_list = (
+    ("uint256", 42),
+    ("int256", -(2 ** 200)),
+    ("int128", -(2 ** 126)),
+    ("address", "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"),
+    ("bytes32", b"deadbeef" * 4),
+    ("bool", True),
+    ("String[10]", "Vyper hiss"),
+    ("Bytes[10]", b"Vyper hiss"),
+)
+
+
+@pytest.mark.parametrize("typ,value", type_list)
```

Comment:
It looks just as good combined like this
```suggestion
@pytest.mark.parametrize(
    "typ,value",
    [
        ("uint256", 42),
        ("int256", -(2 ** 200)),
        ("int128", -(2 ** 126)),
        ("address", "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"),
        ("bytes32", b"deadbeef" * 4),
        ("bool", True),
        ("String[10]", "Vyper hiss"),
        ("Bytes[10]", b"Vyper hiss"),
    ],
)
```